src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/LinkLabel.Link.LinkLabelLinkAccessibleObjectTests.cs

```diff
@@ -0,0 +1,201 @@
+        {
+            using LinkLabel linkLabel = new();
+
+            AccessibleObject expected = linkLabel.AccessibilityObject;
+
+            for (int index = 0; index < 4; index++)
+            {
+                linkLabel.Links.Add(new());
+            }
+
+            for (int index = 0; index < 4; index++)
+            {
+                LinkLabelLinkAccessibleObject linkAccessibleObject = linkLabel.Links[index].AccessibleObject;
+                var actual = (AccessibleObject)linkAccessibleObject.FragmentNavigate(UiaCore.NavigateDirection.Parent);
+
+                Assert.Equal(expected, actual);
+            }
+
+            Assert.False(linkLabel.IsHandleCreated);
+        }
+
+        [WinFormsFact]
+        public void LinkLabelLinkAccessibleObject_Parent_IsExpected()
+        {
+            using LinkLabel linkLabel = new();
+            LinkLabelLinkAccessibleObject linkAccessibleObject = linkLabel.Links[0].AccessibleObject;
+
+            AccessibleObject expected = linkLabel.AccessibilityObject;
+            AccessibleObject actual = linkAccessibleObject.Parent;
+
+            Assert.Equal(expected, actual);
+            Assert.False(linkLabel.IsHandleCreated);
+        }
+
+        [WinFormsFact]
+        public void LinkLabelLinkAccessibleObject_Role_IsExpected()
+        {
+            using LinkLabel linkLabel = new();
+            LinkLabelLinkAccessibleObject accessibleObject = linkLabel.Links[0].AccessibleObject;
+
+            AccessibleRole actual = accessibleObject.Role;
+
+            Assert.Equal(AccessibleRole.Link, actual);
+            Assert.False(linkLabel.IsHandleCreated);
+        }
+
+        [WinFormsTheory]
+        [InlineData(((int)UiaCore.UIA.LegacyIAccessiblePatternId))]
+        [InlineData(((int)UiaCore.UIA.InvokePatternId))]
+        public void LinkLabelLinkAccessibleObject_GetPropertyValue_IsPatternSupported(int patternId)
+        {
+            using LinkLabel linkLabel = new();
+            LinkLabelLinkAccessibleObject accessibleObject = linkLabel.Links[0].AccessibleObject;
+
+            bool actual = (bool)accessibleObject.GetPropertyValue((UiaCore.UIA)patternId);
+
+            Assert.True(actual);
+            Assert.False(linkLabel.IsHandleCreated);
+        }
+
+        [WinFormsFact]
+        public void LinkLabelLinkAccessibleObject_GetPropertyValue_RuntimeIdNotNull()
+        {
+            using LinkLabel linkLabel = new();
+            LinkLabelLinkAccessibleObject accessibleObject = linkLabel.Links[0].AccessibleObject;
+
+            object actual = accessibleObject.GetPropertyValue(UiaCore.UIA.RuntimeIdPropertyId);
+
+            Assert.NotNull(actual);
+            Assert.False(linkLabel.IsHandleCreated);
+        }
+
+        [WinFormsFact]
+        public void LinkLabelLinkAccessibleObject_GetPropertyValue_NameIsExpected_OneLink()
+        {
+            using LinkLabel linkLabel = new();
+            linkLabel.Text = "TestNameLink";
```

Comment:
If you create two or more same constants, keep them in a variable.
PS. use `testName` for the Assert too
```suggestion
            using LinkLabel linkLabel = new();
            
            string testName = "TestNameLink";
            linkLabel.Text = testName ;
```